events/0916_seo.html

```diff
@@ -0,0 +1,178 @@
+
+    <link rel="stylesheet" href="css/reveal.min.css">
+    <link rel="stylesheet" href="css/theme/gdiaa.css" id="theme">
+    <link rel="stylesheet" href="lib/css/rainbow.css">
+    <link rel="stylesheet" href="css/print/pdf.css" media="print">
+
+    <script src="lib/js/head.min.js"></script>
+
+    <!--[if lt IE 9]>
+    <script src="lib/js/html5shiv.js"></script>
+    <![endif]-->
+</head>
+<body>
+    <div class="reveal">
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+
+            <!-- Title -->
+            <section>
+                <img src="images/gdi_logo_badge.png" class="img--bare" height="400" />
+                <div class="box--small">
+                    <h3><span class="blue">Intro to Seach Engine Optimization (SEO)</span></h3>
+                    <h4><span class="green">Workshop</span></h4>
+                    <p class="copy--xsmall">Emily Drier &nbsp; | &nbsp; @emilydrier &nbsp; | &nbsp; @gdiannarbor &nbsp; | &nbsp; #gdia2</p>
+                </div>
+            </section>
+
+            <!-- GDI Welcome -->
+            <section>
+                <h3>Welcome!</h3>
+                <p class="box"><strong>Girl Develop It</strong> is here to empower and assist women interested in getting into and advancing in the tech industry through affordable and accessible programs.</p>
+                <!-- <p class="box"><strong>Girl Develop It</strong> is here to provide affordable programs for adult women interested in learning web and software development in a judgment-free environment.</p> -->
+                <!-- Uncomment for Workshops -->
+                <div class="copy--small">
+                    <h4 class="green">Some "rules"</h4>
+                    <ul>
+                        <li>We are here for you!</li>
+                        <li>Every question is important.</li>
+                        <li>Help each other.</li>
+                        <li><strong>Have fun!</strong></li>
+                    </ul>
+                </div>
+                <p class="box copy--small"><small><strong>Girl Develop It is dedicated to providing a harassment free learning experience for <em>everyone</em>.</strong><br />For more information, see our <a href="http://www.girldevelopit.com/code-of-conduct" target="_blank">Code of Conduct</a>.</small></p>
+            </section>
+
+            <!-- Upcoming Events -->
+            <section>
+                <h3>Upcoming GDI Event</h3>
+                <div class="box">
+                    <h3 class="event__title box--small green">Information as Activism: <br /><small>Using Open Source Tech <br />to Fight Human Trafficking</small></h3>
+                    <h3 class="event__title"><small>Tuesday 9/20 @ Nexient</small></h3>
+                    <p><small>6:30pm - 8:30pm &nbsp; | &nbsp; 1136 Oak Valley Dr</small></p>
+                    <aside class="notes"></aside>
+                </div>
+            </section>
```

Comment:
Can you add Code & Coffee and the newly listed Git workshop as one of these big event slides?